keybase/client
# reset: hook up desktop modal

Issue: Y2K-160

cc @keybase/y2ksquad 

To test, 
```keybase account reset-start YOUR_USERNAME```
and then open the app. Click cancel and see it work.
